Model/TemplateResolver.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
```

Comment:
Could you adjust the header to get rid of unnecessary lines (leave `@author` and `@copyright` only)?

Reply:
done